plugins/techdocs-react/api-report.md

```diff
@@ -0,0 +1,116 @@
+## API Report File for "@backstage/plugin-techdocs-react"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { AsyncState } from 'react-use/lib/useAsync';
+import { ComponentType } from 'react';
+import { CompoundEntityRef } from '@backstage/catalog-model';
+import { Context } from 'react';
+import { Dispatch } from 'react';
+import { Entity } from '@backstage/catalog-model';
+import { Extension } from '@backstage/core-plugin-api';
+import { default as React_2 } from 'react';
+import { SetStateAction } from 'react';
+import { VersionedValue } from '@backstage/version-bridge';
+
+// @alpha
+export function createTechDocsAddon<TComponentProps>(
```

Comment:
Thoughts on calling this `createTechDocsAddonExtension` instead? Just to stick to the `create*Extension` pattern

Reply:
Seems reasonable.  I think we went with `Addon` as a sort of branded `Extension`, and we'll use that in a variety of contexts (documentation, blog posts, etc), but maybe the name of the function doesn't matter too much.